README.md

```diff
@@ -523,6 +523,18 @@ var objectElement = minim.toElement({ foo: 'bar' });
 var keys = objectElement.keys() // ['foo']
 ```
 
+##### delete
+
+The `delete` method deletes a key from the `ObjectElement` instance.
```

Comment:
Ah. Shouldn't the example give that away? All the other examples do the same thing. Let's not confuse the users by introducing an explanation on not supporting elements.